docs/tutorials/solar_flares.ipynb

```diff
@@ -844,45 +1062,80 @@
    "cell_type": "markdown",
    "id": "61d012c3-76f6-41e8-9294-686bad746738",
    "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
     "tags": []
    },
    "source": [
-    "---\n",
-    "<details>\n",
-    "<summary class=\"solution\">\n",
-    "  Solution\n",
-    "</summary>\n",
-    "\n",
-    "<br>\n",
-    "Bin in time and keep the existing spatial binning.\n",
-    "`transpose` is used to put the time dimension on the slider in the plot.\n",
-    "\n",
-    "```python\n",
-    "temporal_and_spatial = spatial.bin(peak_time=200)\n",
-    "to_plot = temporal_and_spatial.hist()\n",
-    "to_plot.transpose(['peak_time', 'y', 'x']).plot(vmin=0.0 * sc.units.one,\n",
-    "                                                vmax=0.66 * sc.units.one,\n",
-    "                                                labels={\n",
-    "                                                    'x': 'x [asec]',\n",
-    "                                                    'y': 'y [asec]'\n",
-    "                                                },\n",
-    "                                                cmap='inferno')\n",
-    "```\n",
-    "\n",
-    "<br>\n",
-    "Or remove the existing binning, yielding a 1D-variable.\n",
-    "\n",
-    "```python\n",
+    "#### Solution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ba688905-28cf-4147-8d87-fa8e81c6d30f",
+   "metadata": {},
+   "source": [
+    "Here we keep the existing binning and plot time slices of the distribution of the events using the `plopp.slicer` tool.\n",
+    "The slicer returns two widgets, a figure and a slider, to change properties of the figure we access the widget at index 0.\n",
+    "Notice that the color scale is kept fixed by setting `vmin` and `vmax` keyword arguments in the call to plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ec0b5a7-ede2-4bdf-b638-708187c5e50a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib widget\n",
+    "import plopp as pp\n",
+    "temporal_and_spatial = spatial.bin(peak_time=10)\n",
+    "p = pp.slicer(\n",
+    "    temporal_and_spatial.hist(),\n",
+    "    keep=['y', 'x'],\n",
+    "    vmin=sc.scalar(0),\n",
+    "    vmax=temporal_and_spatial.hist().max()\n",
```

Comment:
You can use `autoscale='fixed'` to get the same result I think? See https://scipp.github.io/plopp/reference/generated/plopp.slicer.html